hack/windows/Prepare-Node.ps1

```diff
@@ -86,17 +102,23 @@ New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C
 $StartKubeletFileContent = '$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
 $global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")
 
-$netId = docker network ls -f name=host --format "{{ .ID }}"
+$global:containerRuntime = {{CONTAINER_RUNTIME}}
```

Comment:
Why add global:containerRuntime? I saw you used `$ContainerRuntime -eq "docker"` as the condition in line 89.

Reply:
Reminder

Reply:
@NamanAg30 Since you already have a parameter to indicate the testbed runtime, why don't generate different StartKubeletFileContent instead of checking runtime in its content?